moped-etl/arcgis/settings.py

```diff
@@ -1,6 +1,96 @@
 UPLOAD_CHUNK_SIZE = 100
 
-LAYER_IDS = {"points": 0, "lines": 1, "combined": 2}
+LAYER_IDS = {"points": 0, "lines": 1, "combined": 2, "exploded": 3}
+
+EXPLODED_GEOMETRY_SQL_QUERY = """
+SELECT 
+    --ST_GeometryType(dump.geom) AS geometry_type,
+    --dump.path[1] AS point_index, -- ordinal value of the point in the MultiPoint geometry
```

Comment:
Leave these in? They feel useful and show how to produce data that doesn't feel obviously available to me -- especially the second line.